Sage/carbon
# Icon Font Updates

# Icon Font Updated

As well as the Icon font being updated, the correct code added to show on the carbon demo site.

## 1x Updated Icon

The cross icon has been updated to align correctly with the tick in weight.

### Before
![screen shot 2018-08-29 at 11 49 34](https://user-images.githubusercontent.com/2411024/44783245-a3adb000-ab81-11e8-81d2-862fe280a03c.png)

### After
![screen shot 2018-08-29 at 11 44 46](https://user-images.githubusercontent.com/2411024/44783022-faff5080-ab80-11e8-9d4f-a5a62b0d2db1.png)

## 4x New Icons

We have 4 new icons added to the collection. You can see the 16px and 32px versions below.

![new-icons](https://user-images.githubusercontent.com/2411024/44787746-dfe80d00-ab8f-11e8-9aa9-cebb67ed4f4c.png)

# Testing Instructions

Run the carbon site as normal and go to the Style > Icons in the menu. Navigate the list and see the updated and new icons as per the screenshots in this PR.
